sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -181,7 +181,7 @@ Parameter|Description
 `/d:sonar.verbose=true`|[optional] Sets the logging verbosity to detailed. Add this argument before sending logs for troubleshooting.
 `/d:sonar.dotnet.excludeTestProjects=true`|[optional] Excludes Test Projects from analysis. Add this argument to improve build performance when issues should not be detected in Test Projects.
 `/d:<analysis-parameter>=<value>`|[optional] Specifies an additional {instance} [analysis parameter](/analysis/analysis-parameters/), you can add this argument multiple times.
-
+`/s:<custom.analysis.xml>`|[optional] Overrides the `$install_directory/SonarQube.Analysis.xml`. 
```

Comment:
I assume - mostly from the name of the file - this parameter makes sense only for SQ?

Reply:
Based on the content of the default SonarQube.Analysis.xml file, most probably it is for both SQ and SC.